scripts/combine/setupCombineWMass.py

```diff
@@ -164,6 +165,27 @@ def main(args):
     if not args.wlike:
         combine_helpers.add_scale_uncertainty(cardTool, "integrated", single_v_samples, False, pdf=args.pdf, name_append="Z", scetlib=args.scetlibUnc)
 
+    if args.muonScaleVariation == "smearing_weights":
```

Comment:
Could reduce a fair bit of repeated code here by just modifying the hist and axes rather than everything